src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ArrayOfEmbeddedDataNode.cs

```diff
@@ -88,7 +88,7 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
             builder.RequireInitialPointerAlignment();
 
             if (_sorter != null)
-                _nestedNodesList.Sort(_sorter);
+                _nestedNodesList.ParallelSort(_sorter);
```

Comment:
Hmm, wouldn't calling it <code>.MergeSort</code> be a tad more descriptive given how you call it in the implementation?